projects/plugins/jetpack/modules/sso.php

```diff
@@ -639,8 +639,8 @@ public function jetpack_show_connection_status( $val, $col, $user_id ) {
 					),
 					admin_url( 'admin-post.php' )
 				),
-				esc_attr__( 'Invite the user to this site so they can login via SSO.', 'jetpack' ),
-				esc_html__( 'Invite', 'jetpack' )
+				esc_html__( 'Send invite', 'jetpack' ),
+				esc_attr__( 'This user doesn&#8217;t have a WP.com account and, with your current site settings, won&#8217;t be able to log in. Request them to create a WP.com to be able to function normally.', 'jetpack' )
```

Comment:
Seems like a typo here: `...Request them to create a WP.com to be able to function normally.`

Should be `Request them to create a WP.com account to be able to function normally.`?

Reply:
Makes sense 👍 I copied it from the issue and did not double-check.
Good catch!